### PR TITLE
Add support for Debian 12 bookworm

### PR DESCRIPTION
**Pull request by joancafom**
**Description of the change**

Adds support for Debian 12 _Bookworm_, which was [released](https://www.debian.org/News/2023/20230610) a couple of days ago.

It also makes this the default version of minideb.

**Benefits**

Make minideb support the same OS versions as upstream.

**Possible drawbacks**

None

**Applicable issues**

NA

**Additional information**

NA

### CODE REVIEW

**Review by carrodher — APPROVED** [review]
LGTM

File: mkimage [hunk 1]
```diff
@@ -78,7 +78,7 @@ repo_url="http://deb.debian.org/debian"
 sec_repo_url="http://security.debian.org/"
 
 echo -e "deb ${repo_url} $DIST main" > "$rootfsDir/etc/apt/sources.list"
-if [ "$DIST" == "bullseye" ]; then
+if [ "$DIST" == "bullseye" ] || [ "$DIST" == "bookworm" ]; then
```
Comment (carrodher) [inline mkimage:81]: It seems with this new version there is a change in the security repo URL.

While in [Bullseye](https://wiki.debian.org/DebianBullseye#FAQ) is `deb http://deb.debian.org/debian-security bookworm-security main`, in [Bookwork](https://wiki.debian.org/DebianBookworm#FAQ) is `http://deb.debian.org/debian-security`.

With the current logic, the generated entries will be
```
deb http://deb.debian.org/debian bookworm main
deb http://deb.debian.org/debian bookworm-updates main
deb http://security.debian.org/debian-security bookworm-security main
```
while the desired ones according to the new docs are
```
deb http://deb.debian.org/debian bookworm main
deb http://deb.debian.org/debian bookworm-updates main
deb http://deb.debian.org/debian-security bookworm-security main
```

File: debootstrap/bookworm [hunk 2]
```diff
@@ -0,0 +1 @@
+jessie
```
Comment (carrodher) [inline debootstrap/bookworm:1]: This should now point to `buster`

File: mkimage (hunk 1 above)
Comment (joancafom) [reply mkimage:77]: Just being curious, and according to https://wiki.debian.org/SourcesList, I assume they should be interchangeable (that's why it worked). Nevertheless, I will use what they state in the official release page you mentioned.

**Review by carrodher — APPROVED** [review]
LGTM

**Pull request merged**